purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -1481,13 +1505,24 @@ class Purchases @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE) intern
          * @param [networkUserId] User Id that should be sent to the network. Default is the current App User Id
          */
         @JvmStatic
+        @Deprecated(
+            message = "use .set<NetworkId>() functions instead",
+            replaceWith = ReplaceWith(expression = "set<NetworkId>()"),
+            level = DeprecationLevel.WARNING
+        )
         fun addAttributionData(
             data: JSONObject,
             network: AttributionNetwork,
             networkUserId: String? = null
         ) {
-            backingFieldSharedInstance?.postAttributionData(data, network, networkUserId) ?: {
-                postponedAttributionData.add(AttributionData(data, network, networkUserId))
+            backingFieldSharedInstance?.postAttributionData(data, network.convert(), networkUserId) ?: {
```

Comment:
Added these `network.convert()` to convert from the `Purchases.AttributionNetwork` to the one inside `common`